### ISSUE TITLE
\c does not switch to the requested database

### ISSUE THREAD

**Issue by Shambhu-Basnet** [malisper/pgrust#46]
### Environment

- WebAssembly demo

### Summary

The `\c` meta-command appears to ignore the requested database.

### Reproduction

```sql
create database testdb;

\c testdb;
```

### pgrust output

```text
You are now connected to database "postgres" as user "postgres".
```

### PostgreSQL output

```text
You are now connected to database "testdb" as user "postgres".
```

### Expected behavior

The active database should become `testdb`.

Instead, the session reports it is still connected to `postgres`, suggesting that the requested database is not being selected (or the connection status is reported incorrectly).

**Comment by malisper** [comment]
This is now fixed!
